amida-tech/recruitment-registry
# Feature/rr 553

#### What's this PR do?
Updates seed.js for multi-tenant configuration
#### Related JIRA tickets:
553
#### How should this be manually tested?
Make sure with no env var RECREG_DB_SCHEMA everything works as before.  Test with one schema config (RECREG_DB_SCHEMA=schema1).  Test with 2 schemas (RECREG_DB_SCHEMA=schema1~schema2).  Schemas will need to be created manually before seed.js is run. 
#### Any background context you want to provide?
No
#### Screenshots (if appropriate):
NA
